src/app/static/src/tests/components/accessibility.test.ts

```diff
@@ -30,33 +30,33 @@ describe(`accessibility`, () =>{
     it(`renders accessibility tags as expected`, () => {
         const rendered = getWrapper()
         const store = rendered.vm.$store;
-        const h1 = rendered.find("#accessibility-content")
-            .find("h1");
+        const h1 = rendered.findComponent("#accessibility-content")
+            .findComponent("h1");
```

Comment:
There are lots of example of this sort of thing - as you mention in your comment, you shouldn't be using `findComponent` or `findAllComponents` which use css selectors - are you saying you've fixed these in subsequent commits?

Reply:
Yes just in this particular case, I have reverted all the ones that use css selectors back into normal `find`s or `findAll`s